docs/Security-Compliance/Security-Controls/AC/AC-05-Separation-of-duties.md

```diff
@@ -0,0 +1,62 @@
+# Access Control
+## AC-05 - Separation of Duties
+
+The organization:  
+a. Separates [at a minimum, data creation and control, software development and maintenance, and security functions];
+
+b. Documents separation of duties of individuals; and 
+
+c. Defines information system access authorizations to support separation of duties.
+
+### TDP Implementation
+
+**Part a.**  
+Software development and maintenance can be accomplished by a user that has a *superuser* and *staff* flag in the Django Admin  Console (DAC). This also allows the user to login to the DAC and complete all CRUD functions. System owner, also has *superuser* and *staff* flag, and manages the account access. Additionally, System owner provisions a new user account on request.
+
+Data creation and control action can be completed through the TDP application by a user who has been authenticated and is active in the system. These roles include OFA admin and STT users. These users cannot provision new accounts and do not have access to the account management functionality.
+
+**Part b.**    
+**Privileged users:** Users with responsibilities for software development, maintenance, and security functions will have access to all areas of the system. This includes dev and system owner. These users have the *superuser* flag set to true in the DAC. These users can add a new user, edit a user, and perform any account management related controls. All of the actions are logged into the system logs.
+
+**Non-privileged users - OFA Admin:** These are active users that can login but do not have special privileges. This type of user includes OFA admin. These users have the *staff* flag set to true in DAC. These users can upload TANF reports, and replace and download uploaded TANF reports through the TDP interface.
```

Comment:
Part (b) asks for us to document separation of duties of individuals. Do we need to introduce these privileged and non-privileged categories in order to document that? Every new category or word we use to describe user roles adds additional complexity. 

I'm wondering about why this says OFA Admins won't have special privileges. During our 309 demo this week we discussed OFA Admins having access to the Django Admin Console. Isn't that a special privilege that most users of the system won't have?